src/com/owncloud/android/ui/activity/Uploader.java

```diff
@@ -444,6 +485,27 @@ private void populateDirectoryList() {
             mListView.setOnItemClickListener(this);
         }
     }
+    
+    public void startSyncFolderOperation(OCFile folder) {
+        long currentSyncTime = System.currentTimeMillis(); 
+        
+        mSyncInProgress = true;
+        
+        // perform folder synchronization
+        RemoteOperation synchFolderOp = new RefreshFolderOperation( folder,
+                                                                        currentSyncTime, 
+                                                                        false,
+                                                                        false,
+                                                                        false,
+                                                                        getStorageManager(),
+                                                                        getAccount(),
+                                                                        getApplicationContext()
+                                                                      );
+        synchFolderOp.execute(getAccount(), this, null, null);
+        
+        setSupportProgressBarIndeterminateVisibility(true);
```

Comment:
Maybe you can have a look?
I tried to get mProgressBar via mProgressBar = (ProgressBar) findViewById(R.id.progressBar);
But it is not there.

Also the layout looks on a Nexus5 Emulator strange:
![uploader](https://cloud.githubusercontent.com/assets/5836855/10519203/13b07bb0-7365-11e5-939e-c0c3c298d8d3.jpg)